lib/__tests__/_sendEvent.test.ts

```diff
@@ -493,6 +493,117 @@ describe("sendEvents", () => {
         ]
       });
     });
+
+    it("should not be added if authenticatedUserToken is provided", () => {
```

Comment:
Does this need to change in light of the discussion here? https://algolia.atlassian.net/wiki/spaces/EX/pages/4576346340/2023-05-25+Authenticated+user+token+in+Insights+pipeline?focusedCommentId=4746707079

Reply:
Yes, good timing, just pushed the change!